students/Ben_freeman/lesson03/Slicing_lab.py

```diff
@@ -0,0 +1,24 @@
+def swap_fir_la(seq):
+    """Swaps the first and last elements of a finite sequence"""
+    fir=seq[0:1]
+    la=seq[len(seq):len(seq)-2:-1]
+    mid=seq[1:len(seq)-1]
+    return la+mid+fir
```

Comment:
note that this can be just one line: `return seq[-1:] + seq[1:-1] + seq[:1]`

Reply:
Oh that is much cleaner, nice! Thanks for the feedback.